modules/serverless-pingsource-kn.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-pingsource.adoc
+
+[id="serverless-pingsource-kn_{context}"]
+= Using PingSource with the `kn` CLI
+
+The following sections describe how to create, verify and remove a basic PingSource using the `kn` CLI.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+
+. For each set of ping events that you want to request, create a PingSource in the same namespace as the event consumer:
++
+----
+$ kn source ping create test-ping-source --schedule "*/2 * * * *" --data '{"message": "Hello world!"}' --sink svc:event-display
+----
+
+. Check that the controller is mapped correctly by entering the following command and inspecting the output:
++
+----
+$ kn source ping describe test-ping-source
+----
++
+The output should be similar to:
++
+----
+Name:         test-ping-source
+Namespace:    default
+Annotations:  sources.knative.dev/creator=developer, sources.knative.dev/lastModifier=developer
+Age:          15s
+Schedule:     */2 * * * *
+Data:         {"message": "Hello world!"}
+
+Sink:
+  Name:       event-display
+  Namespace:  default
+  Resource:   Service (serving.knative.dev/v1)
+
+Conditions:
+  OK TYPE                 AGE REASON
+  ++ Ready                 8s
+  ++ Deployed              8s
+  ++ SinkProvided         15s
+  ++ ValidSchedule        15s
+  ++ EventTypeProvided    15s
+  ++ ResourcesCorrect     15s
+----
+
+.Verfication steps
+
+You can verify that the Kubernetes events were sent to the Knative event sink by looking at the message dumper function logs.
+By default, a Knative service will be created to receive each new message that is sent. The service will be terminated after 60 seconds.
+The example shown in this guide creates a PingSource that sends a message every 2 minutes, therefore each message that is sent will be received by a newly created pod, which will be terminated after 60 seconds.
+
+. Watch for new pods created:
++
+----
+$ watch oc get pods
+----
+
+. Cancel watching the pods using Ctrl+C, then look at the logs of the created pod:
++
+----
+$ oc get pods
```

Comment:
Now, after changes this first command `oc get pods` is no longer required, as we already checked that pod is there.